lib/tasks/job_runs.rake

```diff
@@ -1,30 +1,32 @@
 namespace :job_runs do
   task run: :environment do
     warn 'Calling job runner. See rails log for output.'
+    @keep_jobs_loop = true
+    @jobs_pid_file = Rails.root.join('tmp', 'job_runs-0.pid')
```

Comment:
Do we want to use Tempfile here?  Can two separate individuals kick off this script concurrently?

Reply:
@brodygov made a similar comment on https://github.com/18F/identity-devops/pull/1492.

Maybe my thinking is old-school here. Here are the 3 options I see: 

1. We could use `kill $(ps aux | pgrep 'job_runs:run')` to stop a previously running process. However, It is my understanding that this is a bad practice and considered less reliable. Process names change when the code changes.

2. We could use `monit procmatch 'job_runs:run'`.  IDP rails devs will have to install/run monit when creating/testing jobs. This is also susceptible to process name changes.

3. We continue using the pidfile. Less susceptible to failure due to process name and code changes. 

Let me know what method you see working the best.